### PR TITLE
Integration test for configuration management

### LINKED ISSUE #1528: Support configuration management on the backend.
### What would you like to be added?

Add configuration management backend services

### Why is this needed?

Enables users to configure plug-ins on the backend.

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Test

**What this PR does / why we need it?**

1、Integrated testing with added configuration management functionality

**Which issue(s) this PR fixes？**

Fixes #1528 

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: .github/actions/scenarios/backend/config/action.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: "Backend Test Use Memory"
```
Comment (lilai23) [inline .github/actions/scenarios/backend/config/action.yml]: action name is not correct

File: sermant-integration-tests/scripts/test_config.py [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+# limitations under the License.
+#
+#
+
+import requests
+import time
+import unittest
+
+
+class TestConfig(unittest.TestCase):
```
Comment (lilai23) [inline sermant-integration-tests/scripts/test_config.py]: file name can be more accurate

File: .github/actions/scenarios/backend/config/action.yml (hunk 1 above)
Comment (hanbingleixue) [reply .github/actions/scenarios/backend/config/action.yml]: done

File: sermant-integration-tests/scripts/test_config.py (hunk 2 above)
Comment (hanbingleixue) [reply sermant-integration-tests/scripts/test_config.py]: done

File: sermant-integration-tests/scripts/test_configuration_management_of_backend.py [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+import time
+import unittest
+
+
+class TestConfig(unittest.TestCase):
+    backend_server_address_use_zookeeper = "http://127.0.0.1:8901"
+    backend_server_address_use_kie = "http://127.0.0.1:8902"
+    backend_server_address_use_nacos = "http://127.0.0.1:8903"
+    request_url_query_configs = "/sermant/configs"
+    request_url_other = "/sermant/config"
```
Comment (daizhenyu) [inline sermant-integration-tests/scripts/test_configuration_management_of_backend.py]: request_path_common

File: sermant-integration-tests/scripts/test_configuration_management_of_backend.py [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+    def config_test_function(self, addConfigParam=None, queryConfigsParam=None, queryConfigParam=None,
+                             updateConfigParam=None, deleteConfigParam=None):
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_zookeeper).get("code") == "00")
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_kie).get("code") == "00")
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_nacos).get("code") == "00")
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_zookeeper).get("code") == "03")
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_kie).get("code") == "03")
+        self.assertTrue(self.add_config(addConfigParam, self.backend_server_address_use_nacos).get("code") == "03")
+        self.assertTrue(
+            len(self.get_configs(queryConfigsParam, self.backend_server_address_use_zookeeper).get("data")) == 1)
```
Comment (daizhenyu) [inline sermant-integration-tests/scripts/test_configuration_management_of_backend.py]: Adding a Test Case That Is Not Matched in Fuzzy Query

File: .github/actions/scenarios/backend/config/action.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
+name: "Test the configuration management of Backend"
+description: "Auto test for Backend"
+runs:
+  using: "composite"
+  steps:
+    - name: Set up JDK ${{ env.javaVersion }}
```
Comment (daizhenyu) [inline .github/actions/scenarios/backend/config/action.yml]: Extract to Common Action

File: .github/actions/scenarios/backend/config/action.yml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+      run: |
+        bash ./sermant-integration-tests/scripts/checkService.sh http://127.0.0.1:8901/sermant/event/webhooks 120
+        bash ./sermant-integration-tests/scripts/checkService.sh http://127.0.0.1:8902/sermant/event/webhooks 120
+        bash ./sermant-integration-tests/scripts/checkService.sh http://127.0.0.1:8903/sermant/event/webhooks 120
+        bash ./sermant-integration-tests/scripts/checkService.sh http://127.0.0.1:8848/nacos/v2/console/namespace/list 20
+    - name: start test
+      shell: bash
+      run: |
+        python -m unittest ./sermant-integration-tests/scripts/test_configuration_management_of_backend.py
+    - name: if failure then upload error log
```
Comment (daizhenyu) [inline .github/actions/scenarios/backend/config/action.yml:53]: stop backend

File: sermant-integration-tests/scripts/test_configuration_management_of_backend.py (hunk 3 above)
Comment (hanbingleixue) [reply sermant-integration-tests/scripts/test_configuration_management_of_backend.py]: done

File: sermant-integration-tests/scripts/test_configuration_management_of_backend.py (hunk 4 above)
Comment (hanbingleixue) [reply sermant-integration-tests/scripts/test_configuration_management_of_backend.py]: done

File: .github/actions/scenarios/backend/config/action.yml (hunk 5 above)
Comment (hanbingleixue) [reply .github/actions/scenarios/backend/config/action.yml]: done

File: .github/actions/scenarios/backend/config/action.yml (hunk 6 above)
Comment (hanbingleixue) [reply .github/actions/scenarios/backend/config/action.yml:53]: done

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**